### PR TITLE
#169900011 Updating README.md

### PR DESCRIPTION
**Pull request by Mastel22**
### What does this PR do?
 -Update README.md
#### Description of Task to be completed?
   - Update REAME.md for better user direction
#### How should this be manually tested?
   -N/A
#### What are the relevant pivotal tracker stories?
[#169900011](https://www.pivotaltracker.com/story/show/169900011)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +30,42 @@ Make company global travel and accommodation easy and convenient for the strong
...
+
+## Docker
+
+### Setting Up Docker
+
+1. Install Docker on your pc using instruction [here](https://docs.docker.com/install/). Make sure it's running well
+
+2. Clone this repo
+
+3. Follow the `.env.example` file to setup your environment and populate with corresponding values
```
Comment (timolinn) [inline README.md:74]: You mean `Copy`?

File: README.md [hunk 2]
```diff
@@ -30,3 +30,42 @@ Make company global travel and accommodation easy and convenient for the strong
...
+
+1. Run `docker-compose up` to create and start containers
+
+2. Run `docker ps` to get container ID
+
+3. Run `docker exec -it <container id> sh` to enter container
+
+4. Run `npm migrate` followed by ` npx sequelize-cli db:seed:all`
+
+5. Run `npm run watch` and test endpoint using postman using route `http://localhost:3000/<endpoind route>`
```
Comment (timolinn) [inline README.md]: Wouldn't running `docker-compose up` start the application server?

**Review by gildniy — DISMISSED** [review]
Looks good to me!

File: README.md (hunk 2 above)
Comment (Mastel22) [reply README.md]: Sorry I delayed. Yeah  it should've been 'build'  instead of 'up'

File: README.md (hunk 1 above)
Comment (Mastel22) [reply README.md:74]: Please check now I think  it's more clear

**Review by timolinn — APPROVED** [review]

**Review by gildniy — APPROVED** [review]
Looks great.

**Review by b0nbon1 — APPROVED** [review]
LGTM

**Pull request merged**